### ISSUE TITLE
Add CutMix transform to torchvision.transforms

### ISSUE THREAD

**Issue by veax-void** [pytorch/pytorch#43123]
## 🚀 Feature

**CutMix** augmentation strategy: patches are cut and pasted among training images where the ground truth labels are also mixed proportionally to the area of the patches.

## Motivation

Its seems like this regularization increasing performance on visual tasks.
![img](https://raw.githubusercontent.com/clovaai/CutMix-PyTorch/master/img1.PNG)

## Pitch

Can you add this transformation into torchvision.transforms?
And if I can somehow help to make it happen, let me know.

## Additional context

Source: [CutMix regularizer implementation for pytorch](https://github.com/clovaai/CutMix-PyTorch)
[**Finished implementation](**https://github.com/hysts/pytorch_cutmix) but they used random and numpy instead of pytorch inside, but they wrap it for pytorch models

**Comment by vfdev-5** [comment]
@veax-void maybe it would be better to post this feature request directly to torchvision repository.
